### DISCUSSION TITLE
Android Application - No Internet Connection

### DISCUSSION THREAD

**Discussion by BrokenPipeAgain** [strongswan/strongswan#2520] [Q&A]
Hello everyone,

I'm a new user of strongswan but I face one major problem. I'm using IkeV2 certificate to connect from my Android, the connection is established but I have no internet connection. When I do an HTTP request or a PING I have to response.
Could you help me to figure out the problem ?

`
connections {
CERT-AUTH-WITH-POOL {
        local_addrs = 172.16.1.37
        remote_addrs = %any
        local {
            auth = pubkey
            certs = serverCert.pem
            id = "server.strongswan.org"
        }
        remote {
            auth = pubkey
        }
        children {
            net {
                local_ts = 0.0.0.0/0
                remote_ts = 0.0.0.0/0
                updown = /usr/local/libexec/ipsec/_updown iptables
                rekey_time = 5400
                rekey_bytes = 500000000
                rekey_packets = 1000000
                esp_proposals = aes192gcm16-ecp384
                mode = transport
                dpd_action = restart
            }
        }
        pools = my-pool
        version = 2
        mobike = no
        reauth_time = 10800
	encap = yes
        proposals = aes192-sha384-ecp384
    }
  }

pools {
    my-pool {
        addrs = 10.0.8.20/32
	dns = 8.8.8.8, 1.1.1.1
    }
}
`

**Comment by BrokenPipeAgain** [comment]
I have added some rules too:
iptables -t nat -A POSTROUTING -s 10.0.3.0/24 -o eth0 -m policy --dir out --pol ipsec -j ACCEPT
iptables -t nat -A POSTROUTING -s 10.0.3.0/24 -o eth0 -j MASQUERADE

**Reply by tobiasbrunner** [reply]
Those rules don't match your IP pool configuration (you used `10.0.8.20/32`  there, not `10.0.3.0/24`). Make sure they do.
